### ISSUE TITLE
Is it possible to change the placeholder text?

### ISSUE THREAD

**Issue by jseparovic** [josdejong/svelte-jsoneditor#89]
Just wondering if it's possible to change the placeholder text?
I couldn't see anything in the docs.

**Comment by jseparovic** [comment]
Yep that’s it

**Comment by josdejong** [comment]
Makes sense to be able to show your own "welcome text" there. Anyone interested in picking this up and think through what would be a nice API? It would be nice if you can specify not just plain text but HTML.

**Comment by jseparovic** [comment]
It would be a nice option if the placeholder text could be an actual JSON document in tree mode, just greyed out...

**Comment by josdejong** [comment]
Moving this idea to the [Discussions > Ideas](https://github.com/josdejong/svelte-jsoneditor/discussions/categories/ideas) section since we are not actively working on it. Help implementing this would be welcome.
